shell/platform/darwin/macos/framework/Source/FlutterMutatorView.mm

```diff
@@ -85,6 +88,28 @@ FlutterRect ToFlutterRect(const CGRect& rect) {
   };
 }
 
+using MutationVector = std::vector<FlutterPlatformViewMutation>;
+
+/// Returns a vector of FlutterPlatformViewMutation object pointers associated with a platform view.
+/// The transforms sent from the engine include a transform from logical to physical coordinates.
+/// Since Cocoa deals only in logical points, this function prepends a scale transform that scales
+/// back from physical to logical coordinates to compensate.
+MutationVector MutationsForPlatformView(const FlutterPlatformView* view, float scale) {
+  std::vector<FlutterPlatformViewMutation> mutations;
+  mutations.reserve(view->mutations_count + 1);
```

Comment:
Couldn't you just reserve it in the constructor?

Reply:
std::vector(size_t) creates a vector of default-initialised objects rather than just reserving space for them.

Reply:
Ahh, OK. Makes sense, then.